huxley/www/js/components/AdvisorProfileView.js

```diff
@@ -133,7 +133,7 @@ class AdvisorProfileView extends React.Component {
     if (
       registration &&
       registration.registration_fee_paid &&
-      registration.delegate_fees_paid == registration.delegate_fees_owed
+      registration.delegate_fees_paid >= registration.delegate_fees_owed
```

Comment:
Do we provide refunds or what not? This is the advisor checklist so probably not the right view but curious if somewhere we have a flag for a yes we are refunding you in the future.

Reply:
We do provide refunds but until they are processed, it's for the treasurer's records that the fees paid might be greater than fees owed.